### PR TITLE
Little Code Cleanup + Automatic Set Date

### PR DESCRIPTION
**Pull request by FR0ST1N**
* Cleanup
* Automatic Date

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -76,4 +77,13 @@ function attachHtml() {
 function setFullYear() {
   var tempDate = new Date();
   document.getElementById("year-element").innerHTML = tempDate.getFullYear();
+}
+
+// Sets random bg color from the array to body
+function setRandomBackgroundColor() {
+  var colors = ["#00796B", "#0288D1", "#051325"];
```
Comment (Gopinath001) [inline js/index.js]: the new colors added makes the view look like this.
pic

Comment (Gopinath001) [reply js/index.js]: you can see the different color shade right?

Comment (Gopinath001) [reply js/index.js]: only prob here!

Comment (FR0ST1N) [reply js/index.js]: it looks fine on my end

Comment (Gopinath001) [reply js/index.js]: can you see the different shade of colors?

Comment (FR0ST1N) [reply js/index.js]: BG It is on your end.. works fine on mine

Comment (Gopinath001) [reply js/index.js]: I checked the pr on chromium. Nothing more.

Comment (FR0ST1N) [reply js/index.js]: seems to be a chromium bug [link](https://stackoverflow.com/questions/39268675/css-div-background-color-bug-on-chrome)

Comment (Gopinath001) [reply js/index.js]: seems to be the case, firefox and even chrome renders it perfectly. but chromium on latest version is rotten.

IMG_20190320_133104
pc

Comment (Gopinath001) [reply js/index.js]: the color which is already present doesnt not create this, :(

**Comment by FR0ST1N** [comment]
Removed random bg color function and set the bg in css to previously what it was

File: js/index.js (hunk 1 above)
Comment (Gopinath001) [reply js/index.js]: I didn't find a workaround to solve this, but definitely a bug on chromium.

**Comment by Gopinath001** [comment]
fixed #1

**Pull request merged**